pom.xml

```diff
@@ -19,7 +19,7 @@
     <properties>
         <revision>0.16</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.235.5</jenkins.version>
+        <jenkins.version>2.277.1</jenkins.version>
```

Comment:
I'm trying to workaround the issue in my local environment since I get some stacktrace errors:

```
WARNING hudson.slaves.NodeProvisioner#lambda: Unexpected exception encountered while provisioning agent
```

So maybe, my machine is too slow that somehow that particular issue happens....

Reply:
![image](https://user-images.githubusercontent.com/2871786/124394289-c9c30980-dcf6-11eb-8649-a114826c46be.png)